ehr/test/src/org/labkey/test/pages/ehr/ParticipantViewPage.java

```diff
@@ -384,11 +384,10 @@ public void select()
             {
                 log("Selecting Report: " + getLabel());
                 scrollIntoView(_el);
-                doAndWaitForRepeatedPageSignal(_el::click, REPORT_TAB_SIGNAL, Duration.ofSeconds(2));
+                doAndWaitForRepeatedPageSignal(_el::click, REPORT_TAB_SIGNAL, Duration.ofSeconds(3));
```

Comment:
Actually that `Duration` parameter isn't the timeout, that's how long it waits for the page signal to stop updating. This needs to use the overload of this method that takes a custom timeout.
```suggestion
                doAndWaitForRepeatedPageSignal(_el::click, REPORT_TAB_SIGNAL, Duration.ofSeconds(3), longWait());
```

Reply:
Thank you